addon/components/file-renderer/template.hbs

```diff
@@ -0,0 +1,2 @@
+<iframe src={{buildMfrUrl download}} width="100%" height=100% scrolling="no" marginheight="0" frameborder="0">
```

Comment:
Also ember lets you [pass DOM attributes directly to components](https://guides.emberjs.com/v2.4.0/components/customizing-a-components-element/#toc_customizing-the-element). In this case `width="100%" height=100% ` don't seem like things we should rigidly impose on consumers. A better api could be:

```html
{{file-renderer download=download width="100%" height="100%"}}
```

You may need to [set the component's `tagName: ''`](http://emberjs.com/api/classes/Ember.Component.html#property_tagName). By default Ember wraps every component in a div. Not 100% were the passed attributes will end